mender/values.yaml

```diff
@@ -239,14 +239,14 @@ api_gateway:
   env:
     SSL: false
   minio:
-    enabled: true
+    enabled: false
     podAnnotations: {}
     url: "http://minio:9000"
     customRule: null
   storage_proxy:
-    enabled: false
-    url: ""
-    customRule: "HostRegexp(`{domain:^artifacts.*$}`)"
+    enabled: true
+    url: "http://seaweedfs-s3:8333"
+    customRule: "PathRegexp(`^/mender-artifact-storage`)"
```

Comment:
Again, I'm not convinced that this is a good default or should belong to an example values file. This makes assumptions about a chart that is not even an explicit dependency/sub-chart.

Reply:
I see your point, but to make it simple for new users, I only found this way. Maybe I should write this in the doc and add a `fail` in the chart if unset? But I'd like a zero brainer setup for starters..

Reply:
![Not great, not terrible](https://tenor.com/view/chernobyl-not-great-not-terrible-its-okay-gif-24540363.gif)